scripts/install_third_party.py

```diff
@@ -140,162 +142,95 @@ def download_and_untar_files(
-    'http://jqueryui.com/resources/download/%s.zip' % JQUERYUI_FILENAME)
-JQUERYUI_THEMES_ZIP_ROOT_NAME = JQUERYUI_FILENAME
-JQUERYUI_THEMES_TARGET_ROOT_NAME = JQUERYUI_FILENAME
-
-CODEMIRROR_REV = '3.19.0'
-CODEMIRROR_ZIP_URL = 'https://github.com/marijnh/CodeMirror/archive/3.19.0.zip'
-CODEMIRROR_ZIP_ROOT_NAME = 'CodeMirror-%s' % CODEMIRROR_REV
-CODEMIRROR_TARGET_ROOT_NAME = 'code-mirror-%s' % CODEMIRROR_REV
-
-UI_CODEMIRROR_REV = '0.1.2'
-UI_CODEMIRROR_ZIP_URL = (
-    'https://github.com/angular-ui/ui-codemirror/archive/src%s.zip'
-    % UI_CODEMIRROR_REV)
-UI_CODEMIRROR_ZIP_ROOT_NAME = 'ui-codemirror-src%s' % UI_CODEMIRROR_REV
-UI_CODEMIRROR_TARGET_ROOT_NAME = 'ui-codemirror-%s' % UI_CODEMIRROR_REV
-
-UI_MAP_REV = '0.5.0'
-UI_MAP_ROOT_NAME = 'ui-map-%s' % UI_MAP_REV
-UI_MAP_ZIP_URL = (
-    'https://github.com/angular-ui/ui-map/archive/v%s.zip' % UI_MAP_REV)
-UI_MAP_ZIP_ROOT_NAME = UI_MAP_ROOT_NAME
-UI_MAP_TARGET_ROOT_NAME = UI_MAP_ROOT_NAME
-
-# ui-utils contains ui-event, which is needed for ui-map.
-UI_UTILS_REV = '0.1.1'
-UI_UTILS_ROOT_NAME = 'ui-utils-%s' % UI_UTILS_REV
-UI_UTILS_ZIP_URL = (
-    'https://github.com/angular-ui/ui-utils/archive/v%s.zip' % UI_UTILS_REV)
-UI_UTILS_ZIP_ROOT_NAME = UI_UTILS_ROOT_NAME
-UI_UTILS_TARGET_ROOT_NAME = UI_UTILS_ROOT_NAME
-
-UI_SORTABLE_REV = '0.12.6'
-UI_SORTABLE_ZIP_URL = (
-    'https://github.com/angular-ui/ui-sortable/archive/src%s.zip'
-    % UI_SORTABLE_REV)
-UI_SORTABLE_ZIP_ROOT_NAME = 'ui-sortable-src%s' % UI_SORTABLE_REV
-UI_SORTABLE_TARGET_ROOT_NAME = 'ui-sortable-%s' % UI_SORTABLE_REV
-
-NG_JOYRIDE_REV = '0.1.11'
-NG_JOYRIDE_ZIP_URL = (
-    'https://github.com/abhikmitra/ng-joyride/archive/%s.zip' % NG_JOYRIDE_REV)
-NG_JOYRIDE_ZIP_ROOT_NAME = 'ng-joyride-%s' % NG_JOYRIDE_REV
-NG_JOYRIDE_TARGET_ROOT_NAME = 'ng-joyride-%s' % NG_JOYRIDE_REV
-
-BOOTSTRAP_REV = '3.3.4'
-BOOTSTRAP_ROOT_NAME = 'bootstrap-%s-dist' % BOOTSTRAP_REV
-BOOTSTRAP_ZIP_URL = (
-    'https://github.com/twbs/bootstrap/releases/download/v3.3.4/%s.zip'
-    % BOOTSTRAP_ROOT_NAME)
-BOOTSTRAP_ZIP_ROOT_NAME = BOOTSTRAP_ROOT_NAME
-BOOTSTRAP_TARGET_ROOT_NAME = 'bootstrap-%s' % BOOTSTRAP_REV
+
+def download_third_party(source_url):
+    """This download all files to the required folders
+    Args:
+      source_url: the URL fof the json file.
+    """
+    manifest_data = return_json(source_url)
+    dependencies = manifest_data['dependencies']
+    for data in dependencies:
+        dependency = dependencies[data]
+        for dependency_id in dependency:
+            dependency_contents = dependency[dependency_id]
+            if dependency_contents.has_key('filesPath'):
+                DEPENDENCY_REV = dependency_contents['version']
+                DEPENDENCY_URL = dependency_contents['filesPath']
+                DEPENDENCY_FILES = dependency_contents['files']
+                FOLDER_NAME = (
+                    dependency_contents['targetDirPrefix'] + DEPENDENCY_REV)
+                if data == 'thirdPartyStatic':
```

Comment:
I suggest having a dict mapping 'thirdPartyStatic' to THIRD_PARTY_STATIC_DIR etc. and getting rid of the if/else.